README.md

```diff
@@ -1,368 +1,257 @@
++ [Webpack optimization functionality](https://github.com/webpack/docs/wiki/optimization)
++ **(Coming Soon)** [React Router](https://github.com/rackt/react-router)
 
-Sponsored by [ShakaCode.com](http://www.shakacode.com/)
+See the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/) for an example of a live implementation and code.
 
-See [Action Plan for v1.0](https://github.com/shakacode/react_on_rails/issues/1). We're ready v1.0!
-
-Feedback and pull-requests encouraged! Thanks in advance! We've got a private slack channel to discuss react + webpack + rails. [Email us for an invite contact@shakacode.com](mailto: contact@shakacode.com).
-
-Supports:
-
-1. Rails
-2. Webpack
-3. React, both v0.14 and v0.13.
-4. Redux
-5. Turbolinks
-6. Server side rendering with fragment caching
-7. react-router for client side rendering (and server side very soon)
-
-## OPEN ISSUES
-1. Almost all the open issues are nice to haves like more tests.
-2. If you want to work on any of the open issues, please comment on the issue. My team is mentoring anybody that's trying to help with the issues.
-3. Longer term, we hope to put in many conveniences into this gem, in terms of Webpack + Rails integration. We're open to suggestions.
-
-## Links
-1. See https://github.com/shakacode/react-webpack-rails-tutorial/ for how to integrate it!
-2. http://www.railsonmaui.com/blog/2014/10/03/integrating-webpack-and-the-es6-transpiler-into-an-existing-rails-project/
-3. http://forum.shakacode.com
-4. If you're looking for consulting on a project using React and Rails, [email us! contact@shakacode.com](mailto: contact@shakacode.com)? You can first join our slack room for some free advice.
-5. We're looking for great developers that want to work with Rails + React with a distributed, worldwide team, for our own
-products, client work, and open source. [More info here](http://www.shakacode.com/about/index.html#work-with-us).
-
-## How is different than the [react-rails gem](https://github.com/reactjs/react-rails)?
-1. `react_on_rails` depends on [webpack](http://webpack.github.io/). `react-rails` integrates closely with sprockets and
-    helps you integrate JSX and the react code into a Rails project.
-2. Likewise, using Webpack as shown in the [react-webpack-rails-tutorial](https://github.com/justin808/react-webpack-rails-tutorial/)
-   does involve some extra setup. However, we feel that tight and simple integration with the node ecosystem is more than
-   worth any minor setup costs.
-3. `react-rails` depends on `jquery-ujs` for client side rendering. `react_on_rails` has it's own JS code that does not
-   depend on jquery.
-
-## Installation Checklist
-1. Include the gems `react_on_rails` and `therubyracer` like [this](https://github.com/shakacode/react-webpack-rails-tutorial/blob/361f4338ebb39a5d3934b00cb6d6fcf494773000/Gemfile#L42) and run `bundle`. Note, you can sustitute your preferable JavaScript engine.
+## Getting Started
```

Comment:
Before this section, we need an overview of the helper.

https://github.com/shakacode/react_on_rails/blob/add-generators/README.md#rails-view-helpers-in-depth

besides what you have there, we need a snippet showing the ERB syntax.

I think we just move that before the generator guide.